ffi/build.rs

```diff
@@ -3,17 +3,25 @@ use std::{env, process::Command};
 fn main() {
     // Pre-check
     println!("cargo:rerun-if-changed=build.rs");
-    println!("cargo:rerun-if-changed=pkg/shadow/eth/receipt.go");
-    println!("cargo:rerun-if-changed=pkg/shadow/eth/proof.go");
-    println!("cargo:rerun-if-changed=pkg/shadow/lock.go");
+    println!("cargo:rerun-if-changed=pkg/shadow/ffi/eth/receipt.go");
+    println!("cargo:rerun-if-changed=pkg/shadow/ffi/eth/proof.go");
     println!("cargo:rerun-if-changed=pkg/shadow/ffi/mod.go");
 
+    // set the env if build failed
+    //env::set_var("CGO_CFLAGS", "-Wno-undef-prefix");
```

Comment:
Why remove?

Reply:
@xiaoch05

Reply:
There is no CGO_CFLAGS in master branch. So why do we add this variable?